tests/unit/test_formatting.py

```diff
@@ -1,22 +0,0 @@
-import black
```

Comment:
Does this mean that we lose the capability to test black formatting via `pytest` and need to use `tox` for that? This does make some sense, since it is not really a python coding test, more a style check. But it is convenient to type `pytest -k format` for a quick test. On the other hand I am also OK with just typing `black --check` manually, so fine with me.

Reply:
Yes, it does, but you can also run `tox -e formatting` and it will run the test. It also splits it out in `travis` so it will be easier to identify.

Reply:
Yes, I like it, tox seems nice.

Reply:
I think `tox -e formatting` will run for all the py files in the repo and not just run the `test_formating` test.  And if that is the case do we still need the `test_formating` test?

Reply:
That´s correct, so `test_formatting` is surpassed by `tox -e formatting`, and I deleted `test_formatting`.